OpenRA.Mods.Common/Traits/Player/PlayerStatistics.cs

```diff
@@ -133,6 +151,46 @@ void INotifyKilled.Killed(Actor self, AttackInfo e)
 				var cost = self.Info.TraitInfo<ValuedInfo>().Cost;
 				attackerStats.KillsCost += cost;
 				defenderStats.DeathsCost += cost;
+
+				var countForArmy = self.Info.TraitInfo<UpdatesPlayerStatisticsInfo>().CountAsArmy;
```

Comment:
While it would have a negligble effect on performance, it is our style convention to cache the traits `Info` object in an `info` field.  Please change `public class UpdatesPlayerStatisticsInfo : TraitInfo<UpdatesPlayerStatistics>` above to `public class UpdatesPlayerStatisticsInfo : ITraitInfo<UpdatesPlayerStatistics>` and override the `Create` method to call a new `public UpdatesPlayerStatistics(UpdatesPlayerStatisticsInfo info)` ctor that assigns info to a `readonly UpdatesPlayerStatisticsInfo info;` for use here and in Crated.